src/pretix/control/forms/filter.py

```diff
@@ -2018,6 +2018,15 @@ class DeviceFilterForm(FilterForm):
         ],
         required=False,
     )
+    state = forms.ChoiceField(
+        label=_('Device Status'),
+        choices=[
+            ('', _('All Devices')),
+            ('active', _('Active Devices')),
+            ('revoked', _('Revoked Devices'))
```

Comment:
```suggestion
        label=_('Device status'),
        choices=[
            ('', _('All devices')),
            ('active', _('Active devices')),
            ('revoked', _('Revoked devices'))
```

I'm not sure which is more correct, but this would be more consistent with the rest of the system